app/controllers/admin/section_controller.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Admin::SectionController < Admin::AdminController
+  def show
+  end
```

Comment:
I think that this empty method can be deleted. It should work as long as this path is defined in routes

Reply:
I wasn't able to remove this, rails complained

Reply:
Thank you for checking and sorry for bad idea :)

Reply:
No probs! It sounded reasonable :)